include/aos/common/config.hpp

```diff
@@ -176,6 +176,13 @@
 #define AOS_CONFIG_TYPES_NODE_TYPE_LEN 64
 #endif
 
+/**
+ * Version max len.
+ */
+#ifndef AOS_CONFIG_TYPES_VERSION_LEN
+#define AOS_CONFIG_TYPES_VERSION_LEN 32
```

Comment:
We could use this const instead of `AOS_CONFIG_TYPES_VENDOR_VERSION_LEN` and `cVendorVersionLen` as well.

Reply:
I'd make this change in a separate PR, this change requires changes in  2 repos: core lib and core api (nanopb templ file)

Reply:
Created a JIRA ticket: https://jiraeu.epam.com/browse/EPMPAOS-5003